src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/dto/IepSummary.kt

```diff
@@ -46,7 +46,7 @@ data class IepSummary(
 
   val daysOnLevel: Int
     get() {
-      if (iepDetails.isEmpty()) throw ValidationException("iepDetails not available")
+      if (iepDetails.isEmpty()) throw IncorrectUsage("iepDetails for booking $bookingId not loaded")
```

Comment:
You seen, this is the problem with raising this exception now.

Exception suggests "Incorrect usage" but is it? The consumer of the API didn't do anything wrong, it just happened to make a request for which one some of the fields couldn't be calculated and it's getting a `400 Bad Request` (I haven't checked this `IncorrectUsage` exception is handled to response `400 Bad Request` but I'm assuming that would be the case).

Reply:
Actually AFAICT [I don't see this exception being handled by Spring Boot](https://github.com/ministryofjustice/hmpps-incentives-api/blob/INC-815-basic-next-review-date/src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/config/HmppsIncentivesApiExceptionHandler.kt), which should result in a `500 Internal Server Error` for the consumer.

I am still not sure we should raise an exception when we cannot calculate one of the fields ([see previous comment](https://github.com/ministryofjustice/hmpps-incentives-api/pull/187#discussion_r988921210))